website/docs/d/cluster.html.markdown

```diff
@@ -102,7 +102,7 @@ In addition to all arguments above, the following attributes are exported:
 
 * `replication_specs` - Configuration for cluster regions.  See [Replication Spec](#replication-spec) below for more details.
 
-
+* `container_id` - The Network Peering Container ID.
```

Comment:
I'd add to this: 
 * `container_id` - The Network Peering Container ID. The id of the container either created programmatically by the user before any clusters existed in the project or when the first cluster in the region (AWS/Azure) or project (GCP) was created.  
@shum correct description in your opinion?

We should also update the container_id variable at https://www.terraform.io/docs/providers/mongodbatlas/r/network_peering.html that it can be obtained from the cluster returned values if the cluster has been created before the peering connection.